daml-lf/interpreter/src/main/scala/com/digitalasset/daml/lf/speedy/Speedy.scala

```diff
@@ -311,7 +312,8 @@ object Speedy {
         checkSubmitterInMaintainers: Boolean,
         compiledPackages: CompiledPackages,
         scenario: Boolean,
-        seedWithTime: Option[(crypto.Hash, Time.Timestamp)] = None,
+        submissionTime: Time.Timestamp = Time.Timestamp.now(),
```

Comment:
Please remove this default value and rather enforce that we pass the right value in.

Reply:
Done.